bridges/zombienet/environments/rococo-westend/bridges_rococo_westend.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
```

Comment:
@serban300 we reference this script in `polkadot-sdk/cumulus/parachains/runtimes/bridge-hubs/README.md` e.g. `./cumulus/scripts/bridges_rococo_westend.sh run-relay`, could you please also fix/check that README.md that it is still working?